kiwiapp-public/src/components/home/FeaturesBanner.tsx

```diff
@@ -0,0 +1,126 @@
+"use client";
+import Image from "next/image";
+
+import { H2, H3 } from "@/components/common/Headers";
+import { Container } from "@/components/common/Container";
+
+import CarrierBrealdown from "public/assets/images/features/carrier-brealdown.png";
+import NewVsRecurring from "public/assets/images/features/new-vs-recurring.png";
+import IncreasedRevenue from "public/assets/images/features/increased-revenue.png";
+import UnpaidCommissionMotive from "public/assets/images/features/unpaid-commission-motive.png";
+import { Button } from "../common/Button";
+import { FeaturesSlider } from "./FeaturesSlider";
+import GreenShade from "public/assets/images/green-shade.png";
+
+const textData = [
+  {
+    color:
+      "bg-clip-text text-transparent bg-gradient-to-b from-white via-white/50 to-transparent",
+    text: "FMO & IMO Owners",
+  },
+  {
+    color: "text-[#18F1A1]",
+    text: "Agency Owners",
+  },
+  {
+    color:
+      "bg-clip-text text-transparent bg-gradient-to-b from-white via-white/50 to-transparent",
+
+    text: "Carriers",
+  },
+  {
+    color:
+      "bg-clip-text text-transparent bg-gradient-to-b from-white via-white/50 to-transparent",
+
+    text: "Independent Agents",
+  },
+];
+
+const imageData = [
+  {
+    image: CarrierBrealdown,
+    alt: "CarrierBrealdown",
+  },
+  {
+    image: NewVsRecurring,
+    alt: "NewVsRecurring",
+  },
+  {
+    image: IncreasedRevenue,
+
+    alt: "IncreasedRevenue",
+  },
+  {
+    image: UnpaidCommissionMotive,
+
+    alt: "UnpaidCommissionMotive",
+  },
+];
+
+export function FeaturesBanner() {
+  return (
+    <section className='  relative flex justify-center flex-col  pb-8  sm:pb-14 lg:pb-20'>
```

Comment:
I'm seeing a bunch of lines with extra spaces in class names. Could you make sure they are fixed?